index.js

```diff
@@ -98,10 +101,13 @@ const copyCacheToModule = async (moduleExecutablePath, platform, revision) => {
 }
 
 module.exports = async ({
-  platform: platform = currentPlatform,
-  revision: revision = '499413',
-  log: log = false
-} = {}) => {
+                          platform: platform = currentPlatform,
+                          revision: revision = '499413',
+                          log: log = false,
+                          installPath: installPathBase = __dirname,
+                          onProgress: onProgress
+                        } = {}) => {
+  const installPath = `${installPathBase}/.local-chromium`
```

Comment:
this isn't what I'd expect, if I set `installPath` that's where it should be downloaded to, not `${installPath}/.local-chromium`. I'd suggest rather to default `installPath` to `${__dirname}/.local-chromium`.

Reply:
I agree, done